templates/data/api-gateway-integration-tests.yaml

```diff
@@ -1,164 +0,0 @@
-templates:
-  - from: generic.tmpl
-    render:
-      - to: ../../prow/jobs/api-gateway/api-gateway-integration-tests.yaml
-        localSets:
-          job_default:
-            securityContext:
-              privileged: false
-              seccompProfile:
-                type: RuntimeDefault
-            optional: "false"
-            labels:
-              preset-debug-commando-oom: "true"
-              preset-kyma-guard-bot-github-token: "true"
-          job_post:
-            slack_channel: goat-int
-          with_ias_credentials:
-            labels:
-              preset-api-gateway-int-test-client-creds: "true"
-          gardener_gcp_job:
-            labels:
-              preset-gardener-gcp-kyma-integration: "true"
-              preset-cluster-version: "true"
-            env:
-              KYMA_PROJECT_DIR: "/home/prow/go/src/github.com/kyma-project"
-          gardener_aws_job:
-            labels:
-              preset-gardener-aws-kyma-integration: "true"
-              preset-cluster-version: "true"
-            env:
-              KYMA_PROJECT_DIR: "/home/prow/go/src/github.com/kyma-project"
-        jobConfigs:
-          - repoName: "kyma-project/api-gateway"
-            jobs:
-              - jobConfig:
```

Comment:
Why do we not have this test and `post-api-gateway-integration-aws` in `api-gateway-manager-integration-tests.yaml`?